### ISSUE TITLE
Issues with sending attachments using SMTP Protocol through Outlook 365 API

### ISSUE THREAD

**Issue by thomascytsang** [WPManageNinja/fluent-smtp#92]
When sending emails without attachments, it delivers. When sending emails with attachments it fails.

Emails keep failing to send via outlook smtp even though they are correct: { "code": 423, "message": "Something with wrong with Outlook API. Please check your API Settings", "errors": [ "Something with wrong with Outlook API. Please check your API Settings" ] }

**Comment by amaurylam** [comment]
Having the same issue on Outlook 365 API but only with attachments > 4MB.
Smaller attachments are successfully delivered.

**Comment by djrm911** [comment]
Yup, looks like there are two separate methods for including attachments depending if they are more or less then 3mbs.

https://learn.microsoft.com/en-us/graph/outlook-large-attachments?tabs=http
